CMakeLists.txt

```diff
@@ -175,12 +178,12 @@ include(cmake/SetupCompilerFlags.cmake)
 include(cmake/SetupUmpireThirdParty.cmake)
 
 if (WIN32)
-  if ((NOT UMPIRE_ENABLE_FILESYSTEM) OR (CMAKE_CXX_STANDARD LESS 17))
```

Comment:
This whole block can be removed now we require C++17, and we can always turn on UMPIRE_ENABLE_FILESYTEM for WIN32 builds